internal/xds/translator/accesslog.go

```diff
@@ -127,7 +126,7 @@ func buildXdsAccessLog(al *ir.AccessLog, forListener bool) []*accesslog.AccessLo
 				GrpcService: &cfgcore.GrpcService{
 					TargetSpecifier: &cfgcore.GrpcService_EnvoyGrpc_{
 						EnvoyGrpc: &cfgcore.GrpcService_EnvoyGrpc{
-							ClusterName: buildClusterNameForOpenTelemetry(otel),
```

Comment:
why is this deleted ? dont we need a unique naming if connecting to different otel collectors ?